packages/mdc-switch/adapter.ts

```diff
@@ -48,4 +48,9 @@ export interface MDCSwitchAdapter {
    * Sets the disabled state of the native HTML control underlying the switch.
    */
   setNativeControlDisabled(disabled: boolean): void;
+
+  /**
+   * Set an attribute value of the native HTML control underlying the switch.
```

Comment:
Make sure you sync-in all the changes from your CL. :)

Reply:
Yes, I remember that MDC Web is not synced yet, so added commit with typo fix. Thank you!

Reply:
Added to PR description and to [commit](https://github.com/material-components/material-components-web/pull/5357/commits/57100507c6372332ed8417daaae30925e160e97c).